docs/observability.md

```
@@ -0,0 +1,178 @@
+# Observability
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)